apps/background/src/app/background-process.ts

```diff
@@ -19,7 +19,10 @@ export function initBackgroundProcess() {
     switchMap(
       ([user, settings]) => {
         if (!settings) NEVER as Observable<IUserSettings>
-        return interval((settings.showCardDurationSeconds + (settings.showCardIntervalDurationMinutes * 60)) * 1000).pipe(
+        const period = (settings.showCardDurationSeconds + (settings.showCardIntervalDurationMinutes * 60)) * 1000;
+        const initialDelay: Date | number = settings.disableUntil?.showWord?.toDate() ?? 0;
```

Comment:
The default value for initialDelay shouldn't be zero but rather period. Otherwise, we'll show the showWordCard/MCQCard component immediately the user updates his settings.

```suggestion
        const initialDelay: Date | number = settings.disableUntil?.showWord?.toDate() ?? period;
```